### PR TITLE
Use Java 15+ multi-line strings where applicable

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java [hunk 1]
```diff
@@ -21,18 +21,15 @@ public final class MacheteFileUtils {
   private MacheteFileUtils() {}
 
   public static String getSampleMacheteFileContents() {
-    // We're deliberately using \n rather than `System.lineSeparator()` here
-    // since it turned out that even on Windows (which generally uses \r\n) IntelliJ expects \n in Color Settings code sample
-    // (and probably in Code Style as well).
-    @SuppressWarnings("regexp") String NL = "\n";
+    return """
```
Comment (PawelLipski) [inline frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java:24]: Still looks as expected:

image

Comment (PawelLipski) [reply frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java:24]: (this is in Code Style settings)

Comment (mkondratek) [reply frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java:24]: great <3

**Review by mkondratek — APPROVED** [review]
finally 💟

**Pull request merged**